test/mako/stub-sidecar/read_results.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+# Copyright 2018 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This scripts helps to read results of mako-stub from http
+
+check_command_exists() {
+  CMD_NAME=$1
+  CMD_INSTALL_WITH=$([ -z "$2" ] && echo "" || printf "\nInstall using '%s'" "$2")
+  command -v "$CMD_NAME" > /dev/null || {
+    echo "Command $CMD_NAME not exists$CMD_INSTALL_WITH"
+    exit 1
+  }
+}
+
+check_command_exists kubectl
+check_command_exists curl
+
+if [[ $# -lt 3 ]]
+then
+  echo "Usage: $0 <mako_stub_pod_name> <mako_stub_namespace> <mako_stub_port> <timeout> <retries> <retries_interval> <out_file>"
+  exit 1
+fi
+
+MAKO_STUB_POD_NAME="$1"
+MAKO_STUB_NAMESPACE="$2"
+MAKO_STUB_PORT="$3"
+TIMEOUT="$4"
```

Comment:
Similar as the above, will there be any error if the script runner does not provide these three args?